assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -149,22 +149,36 @@ def compute_explanations(model_wrapper, data, categorical_features, target_colum
     :type target_column: string
     :param task_type: the task type (regression or classification) of the resulting model
     :type task_type: string
-    :return: explanation scores for the input data
+    :param local_explanations: If local explanations are wanted
+    :type local_explanations: boolean
+    :return: global explanation scores for the input data and, if specified, local explanations scores
     :rtype: list[float]
```

Comment:
wondering if the return type should change in the event of local and global importances being returned ? is it a tuple then ?